python/Sentinel1_Stats/Sentinel1_Stats.ipynb

```diff
@@ -0,0 +1,2372 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "849c4388-8820-4a11-9588-71c797a0a9f1",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "                <script type=\"application/javascript\" id=\"jupyter_black\">\n",
+       "                (function() {\n",
+       "                    if (window.IPython === undefined) {\n",
+       "                        return\n",
+       "                    }\n",
+       "                    var msg = \"WARNING: it looks like you might have loaded \" +\n",
+       "                        \"jupyter_black in a non-lab notebook with \" +\n",
+       "                        \"`is_lab=True`. Please double check, and if \" +\n",
+       "                        \"loading with `%load_ext` please review the README!\"\n",
+       "                    console.log(msg)\n",
+       "                    alert(msg)\n",
+       "                })()\n",
+       "                </script>\n",
+       "                "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# !pip install jupyter-black jupyter\n",
+    "\n",
+    "import jupyter_black\n",
```

Comment:
I wouldn't impose jupyter-black on people who want to execute the notebook

Reply:
removed